package.json

```diff
@@ -304,11 +295,11 @@
         "sinon": "^2.3.7",
         "source-map-loader": "^0.2.4",
         "start-server-and-test": "^1.10.8",
-        "storybook": "^8.1.11",
+        "storybook": "^8.2.4",
         "storybook-react-intl": "^3.0.1",
-        "string-replace-loader": "^2.2.0",
+        "string-replace-loader": "^3.1.0",
         "style-loader": "^1.0.0",
-        "styled-components": "5.0.0",
+        "styled-components": "^5.3.11",
```

Comment:
this needs to be pinned,  one of the minor versions starts to create a dynamic classes that will break the collapsible sidebar snapshots

Reply:
good point, will revert